scripts/compile.ts

```diff
@@ -1,25 +1,23 @@
-import { promises as filesystem } from 'fs'
-import * as path from 'path'
 import { CompilerOutputContract } from 'solc'
 import { ethers } from 'ethers'
 import dotenv from "dotenv";
 import yargs from 'yargs/yargs';
 import { compileContracts, runScript } from './utils';
 
+const fs = require('fs');
+const path = require('path');
```

Comment:
Why the change from ES modules to commonjs?

Reply:
Fixed by https://github.com/protofire/safe-singleton-factory/commit/51f7591668b4a85b6b5c4d693712375258225699